problems/objects.js

```diff
@@ -215,7 +331,36 @@ const mostCommonElement = () => {};
  * @returns {string[]} Elements or their pair values.
  */
 
-const updateList = () => {};
+ const updateList = () => {}
```

Comment:
This one is a little tricky. I want you to think of how we can loop through an array to check is an element is in an object.